graph/formats/gexf12/gexf.go

```diff
@@ -197,7 +197,7 @@ type AttValue struct {
 	EndOpen   string `xml:"endopen,attr,omitempty"`
 }
 
-// EdgeShape holds the visual representation of an edge with associated
+// Edgeshape holds the visual representation of an edge with associated
```

Comment:
This one is unfortunate. I think the comment is correct, but I don't think the type name is wrong enough to justify a potential breakage.